.env.defaults

```diff
@@ -0,0 +1,50 @@
+### Backend Service API
+# Should be set in .env.local
+REPAIRS_SERVICE_API_URL=http://localhost:5101/api
+REPAIRS_SERVICE_API_KEY=
+
+### Single Sign On
+# Should be set in .env
+GSSO_URL=
+HACKNEY_JWT_SECRET=
+GSSO_TOKEN_NAME=token
+
+### Application environment
+NEXT_PUBLIC_ENV_NAME=localdev
+
+### Node environment
+NEXT_PUBLIC_NODE_ENV=development
+NODE_ENV=development
```

Comment:
I had to remove the `NODE_ENV` line in mine because it was affecting the unit tests: https://github.com/LBHackney-IT/bonuscalc-frontend/commit/e15d1664a60c81c6559ecaa191cb363f9f52b482#diff-e9cbb0224c4a3d23a6019ba557e0cd568c1ad5e1582ff1e335fb7d99b7a1055d